### PR TITLE
Gossmap crash fix

### PR DESCRIPTION
**Pull request by rustyrussell**
https://github.com/ElementsProject/lightning/issues/7249

Can't see how this is happening on a fresh gossip store, but avoid crashing if it does, and put in more sanity checks in the only place I can see that we might do this.

### CODE REVIEW

File: contrib/pyln-grpc-proto/pyln/grpc/node_pb2.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: node.proto
-# Protobuf Python Version: 4.25.1
```
Comment (vincenzopalazzo) [inline contrib/pyln-grpc-proto/pyln/grpc/node_pb2.py:4]: I think we have some problems with the versioning of the Python dependencies. However, you did not make any changes that should impact the grpc proto so you can safely remove them

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
Is it possible that we have another crash for the gossip map? e.g: https://github.com/ElementsProject/lightning/issues/7320

BTW except of the grpc python files we are good to go for me

**Comment by endothermicdev** [comment]
Looks reasonable - I hope we get some more insight from this.  Removed the protobuf diff that was failing check-gen-updated.
ACK 6736f6eea4d2ce7dcc934a41082b7d1e77dbd81b

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/7330/commits/6736f6eea4d2ce7dcc934a41082b7d1e77dbd81b

**Pull request merged**